homeassistant/helpers/issue_registry.py

```diff
@@ -112,6 +112,11 @@ def async_get_issue(self, domain: str, issue_id: str) -> IssueEntry | None:
         """Get issue by id."""
         return self.issues.get((domain, issue_id))
 
+    @callback
+    def async_get_issues(self) -> dict[tuple[str, str], IssueEntry]:
+        """Get issue by id."""
+        return self.issues
```

Comment:
Often only functions should be used on objects to get data from them rather than just grabbing into the internal data members. Hence the function (That is used by the template `{{ issues() }}`).